airbyte-integrations/connectors/source-convex/source_convex/spec.yaml

```diff
@@ -0,0 +1,19 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/convex
```

Comment:
should i modify airbyte-config/init/src/main/resources/seed/source_specs.yaml similarly to this? The tutorial didn't mention editing that, but without modifying it I get errors in integration tests running locally.